content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -23,10 +23,13 @@ the orders' respective delivery orders/receipts. The only condition is that all
 be in the *same category* (Unit, Weight, Volume, Length, etc.). For example, a product can have its
 :guilabel:`Unit of Measure` field set to `feet (ft)` and its :guilabel:`Purchase UoM` set to
 `centimeters (cm)`. When a :abbr:`PO (purchase order)` is created for that product, it will list the
-quantity in centimeters. Then, when the abbr:`PO (purchase order)` is confirmed, Odoo automatically
+quantity in centimeters. Then, when the :abbr:`PO (purchase order)` is confirmed, Odoo automatically
 generates a receipt and converts the centimeters to feet. The receipt will list the quantity in
 feet.
 
+.. seealso::
```

Comment:
Moved this up here to guide readers to the Units of measure doc if they're unfamiliar.

Reply:
Also good!